torch/_inductor/pattern_matcher.py

```diff
@@ -300,31 +355,49 @@ def find_anchor_nodes(self, ctx: MatchContext, searched):
             if isinstance(pattern, PatternExpr):
                 for other_node in pattern.find_anchor_nodes(ctx, searched):
                     for node in other_node.users:
-                        if (
-                            node not in searched
-                            and node.op == self.op
-                            and node.target in self.fns_set
-                        ):
+                        if node not in searched and self._match_fns(node):
                             yield node
                             searched.add(node)
 
 
-class CallFunction(_BaseNodeMatch):
+class CallFunction(_TargetArgsExpr):
     """
     Matches a call_function node in the FX graphs: `fns[i](*args, **kwargs)`
     """
 
     op = "call_function"
 
 
-class CallMethod(_BaseNodeMatch):
+class CallMethod(_TargetArgsExpr):
     """
     Matches a call_method node in the FX graphs: `fns[i].method(*args, **kwargs)`
     """
 
     op = "call_method"
 
 
+class CallFunctionVarArgs(_TargetExpr):
```

Comment:
So, I'm guessing this can be used for the split normalization patterns, since we're just looking for a "split" node, with any args?

Reply:
Yes possibly.